backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc.adoc

```diff
@@ -73,10 +72,11 @@ To schedule a backup at specific intervals, enter the `<duration_in_minutes>` in
 ----
 Enter the minutes value between quotation marks (`" "`).
 ====
-
++
```

Comment:
try removing this indent, which looks like it is causing the build to start reading from <2>

Reply:
fixed, thanks :)